tests/tensorflow2/test_keras.py

```diff
@@ -139,6 +139,7 @@ def helper_keras_gradtape(
                     hook.get_collection(cname).save_config = SaveConfig(end_step=0)
 
     opt = tf.keras.optimizers.Adam()
+    hook.wrap_optimizer(opt)
```

Comment:
which case in wrap_optimizer is used when gradtape is used?

Reply:
We need this to support vanilla TF